core/src/main/scala/org/genivi/sota/core/jsonrpc/client.scala

```diff
@@ -78,21 +77,20 @@ object client extends Dynamic {
   }
 
   final case class Request(method: String, params: Json, id: Int) {
-
-    import shapeless._
-    import syntax.singleton._
-    import io.circe.generic.auto._
-
     def run[A](transport: Json => Future[Json])(implicit decoder: Decoder[A], exec: ExecutionContext): Future[A] = {
-      val request = ('jsonrpc ->> "2.0") :: ('method ->> method) :: ('params ->> params) :: ('id ->> id) :: HNil
+      val request =
+        Json.fromFields(List(
+          "jsonrpc" -> "2.0".asJson,
+          "method" -> method.asJson,
+          "params" -> params,
+          "id" -> id.asJson))
```

Comment:
Ah you did write it, I'm not sure why it disappeared though, since I didn't include it. I'll add it.